usb/esp_tinyusb/tusb_msc_storage.c

```diff
@@ -170,6 +173,10 @@ static esp_err_t msc_storage_read_sector(uint32_t lba,
         void *dest)
 {
     assert(s_storage_handle);
+    ESP_RETURN_ON_FALSE(lba < UINT_MAX, ESP_ERR_INVALID_ARG, TAG, "Overflow lba");
```

Comment:
Seems, that for the overflow verification it is better not to verify overflow for uint32_t value here (cause it barely can happened on 32-bit platform) but to take into consideration the following logic: 
- overflow can happened during increasing operation (sum or multiplication), so every operation should be in boundaries after completion. Overflow here would mean, that the result value will be less that both operands.
- for MSC there should be a total num sectors for the media device which we should return during the read capacity operation. Maybe it makes sense to compare boundaries with this value to be sure that all input request at least in the boundaries which device claims.

Reply:
I have removed these checks. The overflow will be handles by `__builtin_umul_overflow()` and `__builtin_uadd_overflow ()`given in [doc](https://gcc.gnu.org/onlinedocs/gcc/Integer-Overflow-Builtins.html)